compose/cli/main.py

```diff
@@ -1312,3 +1321,51 @@ def build_exec_command(options, container_id, command):
     args += [container_id]
     args += command
     return args
+
+
+def has_container_with_state(containers, state):
+    for container in containers:
+        states = {
+            'running': container.is_running,
+            'stopped': not container.is_running,
+            'paused': container.is_paused,
+        }
+        if state not in states:
+            raise UserError("Invalid state: %s" % state)
+        if states[state]:
+            return True
+    return False
+
+
+def filter_services(filters, services, project):
+    def should_include(service):
+        for f in filters:
+            if f == 'status':
```

Comment:
rather than `status`, this should be called `state` to match the terminology used elsewhere in the application and the Docker ecosystem.

Reply:
Actually, the corresponding filter in `docker ps` is called `status`, too: https://docs.docker.com/engine/reference/commandline/ps/#filtering